app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/aboutapp/AboutAppDestination.kt

```diff
@@ -0,0 +1,170 @@
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.google.accompanist.web.WebView
+import com.google.accompanist.web.rememberWebViewState
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.normal.ChevronRight
+import com.hedvig.android.core.ui.appbar.TopAppBarWithBack
+import com.hedvig.app.BuildConfig
+import com.hedvig.app.feature.embark.ui.MemberIdViewModel
+import hedvig.resources.R
+
+private const val licensesUrl = "file:///android_asset/open_source_licenses.html"
+
+@Composable
+internal fun AboutAppDestination(
+  viewModel: MemberIdViewModel,
+  onBackPressed: () -> Unit,
+  isProduction: Boolean,
+) {
+  val memberIdState by viewModel.state.collectAsStateWithLifecycle()
+  val memberId: String? by remember {
+    derivedStateOf {
+      when (val latestMemberIdState = memberIdState) {
+        is MemberIdViewModel.State.Success -> latestMemberIdState.id
+        else -> null
+      }
+    }
+  }
+
+  var showWebView by remember { mutableStateOf(false) }
+
+  AboutAppScreen(
+    memberId = memberId,
+    onBackPressed = onBackPressed,
+    showOpenSourceLicenses = { showWebView = true },
+    isProduction = isProduction,
+  )
+
+  if (showWebView) {
+    val state = rememberWebViewState(licensesUrl)
+    WebView(state = state)
+  }
+}
+
+@Composable
+private fun AboutAppScreen(
+  memberId: String?,
+  onBackPressed: () -> Unit,
+  showOpenSourceLicenses: () -> Unit,
+  isProduction: Boolean,
+) {
+  Surface(
+    color = MaterialTheme.colorScheme.background,
+    modifier = Modifier.fillMaxSize(),
+  ) {
+    Column(
+      Modifier
+        .fillMaxSize()
+        .verticalScroll(rememberScrollState()),
+    ) {
+      TopAppBarWithBack(
+        onClick = onBackPressed,
+        title = stringResource(R.string.PROFILE_ABOUT_APP_TITLE),
+        contentPadding = WindowInsets.safeDrawing
+          .only(WindowInsetsSides.Horizontal + WindowInsetsSides.Top)
+          .asPaddingValues(),
+      )
+      Spacer(Modifier.height(16.dp))
+      Column {
+        Row(
+          horizontalArrangement = Arrangement.SpaceBetween,
+          verticalAlignment = Alignment.CenterVertically,
+          modifier = Modifier
+            .fillMaxWidth()
+            .padding(16.dp),
+        ) {
+          Text(stringResource(id = R.string.PROFILE_ABOUT_APP_MEMBER_ID))
+          CompositionLocalProvider(LocalContentColor provides MaterialTheme.colorScheme.onSurfaceVariant) {
+            Text(memberId ?: "")
+          }
+        }
+        Row(
+          horizontalArrangement = Arrangement.SpaceBetween,
+          verticalAlignment = Alignment.CenterVertically,
+          modifier = Modifier
+            .fillMaxWidth()
+            .padding(16.dp),
+        ) {
+          Text("Version")
```

Comment:
We don't need this to be localized I suppose right?

Reply:
Added string resource just in case 👍